library/psa_crypto.c

```diff
@@ -4229,10 +4229,9 @@ static psa_status_t its_to_psa_error( psa_its_status_t ret )
         case PSA_ITS_ERROR_INSUFFICIENT_SPACE:
             return( PSA_ERROR_INSUFFICIENT_STORAGE );
 
-        case PSA_ITS_ERROR_INVALID_KEY:
-        case PSA_ITS_ERROR_OFFSET_INVALID:
+        case PSA_PS_ERROR_OFFSET_INVALID:
         case PSA_ITS_ERROR_INCORRECT_SIZE:
-        case PSA_ITS_ERROR_BAD_POINTER:
+        case PSA_ITS_ERROR_IVALID_ARGUMENTS:
```

Comment:
Looks like a typo that was copied over from the PSA ITS repo, "IVALID" should be "INVALID". It would be good to correct the spelling before it gets set in multiple repos.